### ISSUE TITLE
NTR: Head imaging: cerebral contusion (Lab: radiology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9020]
New term request Head imaging: cerebral contusion (seidel_190211162737): MRI; CT; intracranial; bruising; bruise; brain  UMLS: 0149844

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000141
- label: Cerebral contusion
- parents: [Abnormal cerebral morphology (HP:0002060)](https://hpo.jax.org/app/browse/term/HP:0002060)
- pmid: PMID:32965818
- def: A contusion is a region of injured tissue in which blood capillaries have been ruptured without laceration (bruise). Brain contusions are bruises of the cortical surface that damage the surface from the outside inward, producing disruption of tissue and vessels. The term cerebral contusion describes the pathology of focal necrosis and hemorrhage, typically observed in older children, involving particularly cerebral cortex and subcortical white matter. Such lesions are usually found in coup and contrecoup, as well as inferior orbital, frontal, and temporal locations. Cerebral contusions cause permanent damage to tissues of the cerebrum. The etiology of cerebral contusion is trauma to the head.
- comment: Cerebral contusion is the classic example of focal traumatic brain injury (TBI). In the pre-CT era, cerebral contusion could be diagnosed only in the operating room during craniotomy or at the autopsy table. Initially, the CT may show isodense or hypodense areas, sometimes mixed-density lesions commonly surrounded by perilesional hypodense areas, but later more surrounding edema with progression over time. Multiple focal contusions may have a salt and pepper appearance. The bone window is used for the evaluation of skull fractures. Brain magnetic resonance imaging (MRI) is not indicated in the acute management of TBI. If acutely done, there are isointense to hyperintense areas on T1-weighted images and hyperintense on T2-weighted images. Fluid-attenuated inversion recovery images demonstrate petechial hemorrhages that are isointense to the brain. The MRI can show ischemic areas and changes from diffuse axonal injury. It can be used later for cognitive problems.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9020
- synonyms:
